float32 servo_angle